doc/rtd/topics/events.rst

```diff
@@ -0,0 +1,81 @@
+.. _events:
+
+******************
+Events and Updates
```

Comment:
This doc could use some work to make it a little more accessible for cloud-init users.